<!DOCTYPE html>
<html lang="en">
<head>
<!-- basic -->
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<!-- mobile metas -->
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="viewport" content="initial-scale=1, maximum-scale=1">
<!-- site metas -->
<title>Analyzing Microbiomes</title>
<meta name="keywords" content="">
<meta name="description" content="">
<meta name="author" content="">
<!-- bootstrap css -->
<link rel="stylesheet" href="css/bootstrap.min.css">
<!-- style css -->
<link rel="stylesheet" href="css/style.css">
<!-- Responsive-->
<link rel="stylesheet" href="css/responsive.css">
<!-- fevicon -->
<link rel="icon" href="images/fevicon.png" type="image/gif" />
<!-- Scrollbar Custom CSS -->
<link rel="stylesheet" href="css/jquery.mCustomScrollbar.min.css">
<link rel="icon" href="images/icon.jpg">

<!-- Tweaks for older IEs-->
<!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script><![endif]-->
</head>
<!-- body -->
<body class="main-layout">
<!-- loader  -->
<div class="loader_bg">
  <div class="loader"><img src="images/loading.gif" alt="#" /></div>
</div>
<!-- end loader --> 
<!-- header -->
<header> 
  <!-- header inner -->
  <div class="head-top">
    <div class="container">
      <div class="row">
        <div class="col-xl-4 col-lg-4 col-md-4 col-sm-4">
          <div class="email"> <a style="color:transparent;" href="#">Email : fascettij21@bcdschool.org</a> </div>
        </div>
        <div class="col-xl-4 col-lg-4 col-md-4 col-sm-4">
          <!--
          <div class="icon"> <i> <a href="#"><img src="icon/facebook.png"></a></i> <i> <a href="#"><img src="icon/Twitter.png"></a></i> <i> <a href="#"><img src="icon/linkedin.png"></a></i> <i> <a href="#"><img src="icon/google+.png"></a></i> </div>
          -->
        </div>
        <div class="col-xl-4 col-lg-4 col-md-4 col-sm-4">
          <!--
          <div class="contact"> <a href="#">Contact :  +71  78908540</a> </div>
          -->
        </div>
      </div>
    </div>
  </div>
  <div class="container">
    <div class="row">
      <div class="col-xl-3 col-lg-3 col-md-3 col-sm-3 col logo_section">
        <div class="full">
          <div class="center-desk">
            <!--
            <div class="logo"> <a href="index.html"><img src="images/logo.jpg" alt="#"></a> </div>
            -->
          </div>
        </div>
      </div>
      <div class="col-xl-9 col-lg-9 col-md-9 col-sm-9">
        <div class="menu-area">
          <div class="limit-box">
            <nav class="main-menu">
              <ul class="menu-area-main">
                <li> <a href="index.html">Home</a> </li>
                <li> <a href="what-are-microbiomes.html">What are Microbiomes?</a> </li>
                <li class="active"> <a href="analyze.html">Analyze</a> </li>
                <li> <a href="impact.html">Impact</a> </li>
              </ul>
            </nav>
          </div>
        </div>
      </div>
    </div>
  </div>
  <!-- end header inner --> 
</header>
<!-- end header -->

<div class="Team-bg">
  <div class="container">
    <div class="row">
      <div class="col-xl-12 col-lg-12 col-md-12 col-sm-12">
        <div class="abouttitle">
          <h2>Analyzing a Microbiome</h2>
        </div>
      </div>
    </div>
  </div>
</div>

<!-- state abouts -->
<div class="abouts"> <dir class="abouts-bg">
  <div class="container">
    <div class="row">
      
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <h3>How do you <strong class="cur">analyze</strong> a microbiome?</h3>
          <p>
            Microbiomes are composed of many different species of microorganisms, and each microbiome (both intra- and inter-person) can be vastly different from another. Thus, being able to determine what an individual’s microbiome contents are is very important. To do this, the genetic code of the microbes can be sequenced. From a sample, scientists can <a target="_blank" style="font-weight:450;color:#a02f16;" href="https://www.labome.com/method/DNA-Extraction-and-Purification.html">extract the genetic code</a> and then sequence it to know what the <a style="font-weight:450;color:#a02f16;" href="what-is-dna.html#dna">DNA</a> code is. Typically, when researchers want a microbiome sample sequenced, they will pay a third party company to sequence the sample and receive electronic files with the DNA code from the sample. There are different ways to sequence a microbiomic sample: read below to learn more about 16S rRNA sequencing, shotgun metagenomics, and metatranscriptome sequencing. Most of the time, researchers will choose one method over another either because of their budget or depending on the goal of the study, but there are pros and cons to each approach.
          </p>
        </div>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <br><br><br><br><br><br><br><br><br><br><br><br>
          <a href="analyze.html#cites">
            <figure><img title="Image 1" style="margin-left:60px" src="images/microbiome-contents.jpg" alt="img"/>
              <p style="text-align:center;"><i>
                <!--Image 1: Human microbiome contents.-->
              </i></p>
            </figure>
          </a>
        </div>
      </div>
      
    </div>
  </div>
  </dir> 
</div>

<div class="abouts"> <dir class="abouts-bg">
  <div class="container">
    <div class="row">
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <br><br>
          <a href="#cites">
            <figure><img title="Image 2" src="images/16s-sequencing.png" alt="img"/></figure>
          </a>
          <a href="#cites">
            <p>
              <i>
                Image 2: 16S gene with hypervariable regions.
              </i>
            </p>
          </a>
        </div>
        <div class="abouts-us">
          <br><br>
          <figure><a href="#cites"><img src="images/16s-figure.png" alt="img"/></a></figure>
          <p>
            <i>
              Image 3: figure from <a target="_blank" style="font-weight:450;color:#a02f16;" href="https://doi.org/10.1038/nbt.2676">Langille et al.</a> analyzing microbiomic 16S data
            </i>
          </p>
        </div>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <h3 style="text-transform:inherit;"><strong class="cur">16S rRNA</strong> SEQUENCING</h3>
          <p>
            The 16s <a style="font-weight:450;color:#a02f16;"href="what-is-dna.html#rna">rRNA</a> gene is the most common gene sequenced to identify bacteria and other microorganisms. There are other genes that are in microbes, but the 16s rRNA gene is considered the most conserved throughout species, meaning it is one of the most reliable to identify the parent organism. The 16s rRNA gene is made of nine hypervariable regions which are unique to particular species. When performing 16s rRNA sequencing, two hypervariable regions are usually chosen to identify the species (see Image 2). The hypervariable regions chosen are given <a target="_blank" style="font-weight:450;color:#a02f16;" href="https://www.illumina.com/content/dam/illumina-marketing/documents/products/whitepapers/archerdx-molecular-barcode-whitepaper.pdf">molecular barcodes</a> so computational methods can later detect possible errors and identify sequences easily. The sequencing method uses <a target="_blank" style="font-weight:450;color:#a02f16;" href="https://www.genome.gov/about-genomics/fact-sheets/Polymerase-Chain-Reaction-Fact-Sheet">polymerase chain reaction</a> (PCR) <a href="#footnotes" style="font-weight:450;">[a]</a> to amplify the selected range of DNA in the sample to a detectable level. Typically, <a target="_blank" style="font-weight:450;color:#a02f16;" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3841808/">next generation sequencing</a> (NGS), a modern and rapid method for DNA/RNA analysis, is used to sequence the samples. Computational methods are then used against a reference database of known species’ genomes to determine the contents of the sample. Finally, conclusions can be made from 16S-sequenced data (see Image 3) drawing conclusions in a research study.  
          </p>
        </div>
      </div>
      
      
    </div>
  </div>
  </dir> 
</div>

<div class="abouts"> <dir class="abouts-bg">
  <div class="container">
    <div class="row">
      
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <h3><strong class="cur">Shotgun Metagenomic</strong> Sequencing</h3>
          <p>
            The shotgun metagenomic sequencing method sequences all of the DNA in the sample, then attempts to assign specific parts of the sequence to particular species from a known database. Compared to trying to assign a region from a species from a relatively small portion of the genome in 16S, metagenomic sequencing considers the whole genome. Scientists do this by using NGS to splice the samples’ genome into readable lengths and associate the fragments to a reference genome (see Image 3). This method comes with pros and cons. On the plus side, it tends to be more reliable and accurate. But on the other hand, it is significantly more expensive. Depending on the study done, researchers may opt for one over the other because of price. Though, the cost of shotgun metagenomic sequencing has been decreasing over time, which may cause an increase in popularity.
          </p>
        </div>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <br><br><br><br><br>
          <figure><a href="#cites"><img style="height:414px;width:212px;margin-left:30%;" src="images/metagenomics-img4.jpg" alt="img"/></a></figure>
          <p style="text-align:center;margin-left:10%;margin-right:10%;">
            <i>
              Image 4: DNA fragments (short pieces) are compared against the reference genome (solid line) during read classification to indentity sample contents.
            </i>
          </p>
        </div>
      </div>
      
    </div>
  </div>
  </dir> 
</div>

<div class="abouts"> <dir class="abouts-bg">
  <div class="container">
    <div class="row">
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <br><br><br><br><br><br><br><br>
          <figure><a href="#cites"><img src="images/chem-signal.jpg" alt="img"/></a></figure>
          <p style="text-align:center;margin-left:10%;margin-right:10%;">
            <i>
              Image 5: Chemical signaling is a communication method that cells and microorganism can use to interact with one another.
            </i>
          </p>
        </div>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <h3><strong class="cur">Metatranscriptome</strong> Sequencing</h3>
          <p>
            This type of sequencing is foundationally different from both 16S and shotgun metagenomics, which measure the composition of the microbiome in a taxonomic- and census-style. Metatranscriptome sequencing, however, aims to analyze the microbiome sample for the organisms that are active. For background, microorganisms in the gut microbiome have a connection and interact with the host organism with <a target="_blank" style="font-weight:450;color:#a02f16;" href="https://doi.org/10.3410/B3-4">chemical signals</a> (see Image 5). Another way that the microorganisms are active is by producing proteins. To do this, they need to go through a process of protein synthesis – one step in which is to produce <a style="font-weight:450;color:#a02f16;"href="what-is-dna.html#rna">RNA</a>. Metatranscriptomic sequencing, rather than measuring the DNA, measures this RNA to identify the active microorganisms. This method of sequencing is relatively new, but opens a lot of doors to learning more about microbiomes. A company called <a  target="_blank" style="font-weight:450;color:#a02f16;" href="https://www.viome.com/">Viome</a> specializes in this type of sequencing. <!--Learn more about one of their founders Dr. Momo Vuyisich’s opinions on the benefits of metatranscriptomic sequencing <a style="font-weight:450;color:#a02f16;" href="momo.html">here</a>. -->
          </p>
        </div>
      </div>
      
      
    </div>
  </div>
  </dir> 
</div>

<div class="abouts"> <dir class="abouts-bg">
  <div class="container">
    <div class="row">
      
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <h3><strong class="cur">Computational</strong> Methods</h3>
          <p>
            Computational power is crucial to advancement in the field of microbiomes; samples are useless if researchers have no way to tell what is in them. One of the most limiting factors in modern computational methods of microbiomic (and general genomic) analysis is the inability to process the sheer amount of data that is available. For example, the <a target="_blank" style="font-weight:450;color:#a02f16;" href="ecoli-genome.txt"> file for the genome of escherichia coli</a> <a href="#cites" style="font-weight:450;">[1]</a> <a href="#footnotes" style="font-weight:450;">[b]</a> more commonly known as e. coli, is over 66,000 lines with 70 nucleotides-letters in each line (see Image 6), and an average microbiome has many trillions of microbes in it. <a href="#cites" style="font-weight:450;">[2]</a> Though a single sample will not have every single microorganism in it, the amount of data is extremely large; there must be significant computational power to sift through all of the data and analyze it. This is part of the reason 16S rRNA sequencing is cheaper than shotgun metagenomic sequencing, since the former only does analysis on a particular gene for each organism and the latter sequences the entire genome.
            <br><br>
            There are bioinformatic <a href="#footnotes" style="font-weight:450;">[c]</a> pipelines <a href="#footnotes" style="font-weight:450;">[d]</a> that are free to download like <a target="_blank" style="font-weight:450;color:#a02f16;" href="http://qiime.org/">QIIME</a> (pronounced <i>chime</i>) and <a target="_blank" style="font-weight:450;color:#a02f16;" href="https://mothur.org/">Mothur</a> (see Images 7 and 8), and each source has its own unique procedure. Once the sequences are acquired from samples, the computational workflow for analyzing a microbiome is different for each of 16S, shotgun metagenomics, metatranscriptomics. Whichever bioinformatic tool is used has a specific pipeline and series of steps to demultiplex data <a href="#footnotes" style="font-weight:450;">[e]</a> and align it using dynamic programming <a href="#footnotes" style="font-weight:450;">[g]</a>. For 16S sequencing, the hypervariable regions are cross-referenced to a database of known species to determine the contents of the sample. The computer program used needs to be powerful enough to search for near-matches, not just identical matches. Because it is unlikely that the reference gene sequences will be exactly the same as the one in the sample. This is most probable to happen either because the gene of the sequenced organism itself had a naturally occurring mutation <a href="#footnotes" style="font-weight:450;">[f]</a> or there was an error during the sequencing process – it is widely accepted that sequencing is >97% accurate. <a href="#cites" style="font-weight:450;">[3]</a> Since a difference between the reference and unknown sequence is almost guaranteed, it is important that the computer program compensate for that. On the other hand, if the program is too lax in how many differences it allows, it will falsely classify certain 16S genes as a species. There are algorithms that attempt to diminish the computational power needed to do this analysis called dynamic programming <a href="#footnotes" style="font-weight:450;">[g]</a>. For metagenomic and metatranscriptomic sequencing, similar methods are used but on a different scale. The former analyzes the entire genome of the microorganisms and the latter analyzes the present RNA in a sample. While these methods have made these processes significantly less time and power consuming than brute-force methods, large datasets continue to pose a challenge for microbiome analysis.
          </p>
        </div>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <br><br><br><br><br><br><br><br><br><br><br>
          <figure><img src="images/ecoli-genome.png" alt="img"/>
            <p style="text-align:center;"><i>
              Image 6: The beginning of e coli's genome in the form of a <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&PAGE_TYPE=BlastDocs&DOC_TYPE=BlastHelp" target="_blank" style="font-weight:450;color:#a02f16">.fasta</a> file.
            </i></p>
          </figure><br><br><br><br><br><br><br>
          <figure><img style="margin-left:15%;" src="images/qiime.png" alt="img"/>
            <p style="text-align:center;"><i>
              Image 7: Qiime logo.
            </i></p>
          </figure>
          <br><br><br>
          <figure><img style="width:50%;margin-left:25%;" src="images/mothur.png" alt="img"/>
            <p style="text-align:center;"><i>
              Image 8: Mothur logo.
            </i></p>
          </figure>
        </div>
      </div>
      
    </div>
  </div>
  </dir> 
</div>
<br><br>
<div class="about-bg">
  <div class="container">
    <div class="row">
       <div class="col-xl-12 col-lg-12 col-md-12 col-sm-12">
        <div class="abouttitle">
         <h2>What's next?</h2>
       </div>
        
       </div>
    </div>

  </div>

</div>
<p style="text-align:center;font-size:20px;">
  So what? Find out why microbiomes are important <a href="impact.html" style="color:#a02f16;">here</a>.
</p>
<br><br>
<div class="Currency">
  <div class="container">
<div class="row">
      <div class="col-md-12">
        <div class="titlepage">
          
          <span><img src="images/boder.png" alt="img"/> </span> </div>
      </div>
    </div>
  </div>
</div>



<div class="Request">
  <div class="container">
    <div class="row">
      <div class="col-md-12">
        <div class="titlepage3">
          <h2 id="footnotes">Footnotes</h2>
        </div>
      </div>
    </div>
    <div class="row">
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="foota">
        <b>[a]</b> Polymerase chain reaction (PCR) is a method of replicating specific portions of DNA. Samples undergo a series of heat cycles which activate certain processess to be executed that replicate the selected DNA designated by primers. Learn more about PCR <a target="_blank" style="font-weight:450;color:#a02f16;" href="https://www.sciencelearn.org.nz/resources/2347-what-is-pcr">here</a>. PCR has gained more public attraction because it is one of the most reliable methods to use for a diagnosis of the novel COVID-19. Learn more about PCR COVID-19 tests <a target="_blank" style="font-weight:450;color:#a02f16;" href="https://www.fda.gov/consumers/consumer-updates/coronavirus-disease-2019-testing-basics">here</a>.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="footb">
        <b>[b]</b> E coli (Escherichia coli) is a common bacteria that typically is found in the intestines of animals but is frequently used in lab experiments due to its fast replication period and high protein expression levels.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="footc">
        <b>[c]</b> Bioinformatics is a field of science combining computer science and biology.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="footd">
        <b>[d]</b> A bioinformatic pipeline is a computational process used to analyze data (Ex. microbiomic bioinformatic pipeline).
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="foote">
        <b>[e]</b> To demultiplex data, it is organized from a jumbled mess of DNA code to organized data which can be worked with more easily.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="foote">
        <b>[f]</b> Mutation: A change in genetic code, most frequently during DNA replication.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="footf">
        <b>[g]</b> Dynamic programming is a recursive algorithm frequently used in bioinformatic studies. Typically with the input of a .fasta file, it outputs a comparison of two or more sequences in similarities. Learn more <a target="_blank" style="font-weight:450;color:#a02f16;" href="https://jeffe.cs.illinois.edu/teaching/algorithms/book/03-dynprog.pdf">here</a>.
      </div>

      
      

  </div>
</div>
</div>

<div class="Request">
  <div class="container">
    <div class="row">
      <div class="col-md-12">
        <div class="titlepage3">
          <h2 id="cites">Sources and links</h2>
        </div>
      </div>
    </div>
    <div class="row">
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="image1">
          <b>[Image 1]</b> 
          <a href="https://www.niehs.nih.gov/health/topics/science/microbiome/index.cfm" target="_blank" style="font-weight:450;">https://www.niehs.nih.gov/health/topics/science/microbiome/index.cfm
        </a>
        
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="image2">
          <b>[Image 2]</b> 
          <a href="https://youtu.be/kXYJ_Dc9qcc?t=68" target="_blank" style="font-weight:450;">https://youtu.be/kXYJ_Dc9qcc?t=68
        </a>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
          <b>[Image 3]</b> Langille, M., Zaneveld, J., Caporaso, J. <i>et al.</i> Predictive functional profiling of microbial communities using 16S rRNA marker gene sequences. <i>Nat Biotechnol</i> 31, 814–821 (2013). <a target="_blank" style="font-weight:450;" href="https://doi.org/10.1038/nbt.2676">https://doi.org/10.1038/nbt.2676</a>.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="image2">
          <b>[Image 4]</b> 
          <a href="https://www.eurofinsgenomics.eu/en/eurofins-genomics/material-and-methods/metagenome-analysis/" target="_blank" style="font-weight:450;">https://www.eurofinsgenomics.eu/en/eurofins-genomics/material-and-methods/metagenome-analysis/
        </a>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="image2">
          <b>[Image 5]</b> 
          <a href="https://www.staff.ncl.ac.uk/t.r.cheek/cellcomm.htm" target="_blank" style="font-weight:450;">https://www.staff.ncl.ac.uk/t.r.cheek/cellcomm.htm
        </a>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="cite1">
          <b>[1]</b> <i>Escherichia coli str. K-12 substr. MG1655, complete genome.</i> NCBI. <a target="_blank" style="font-weight:450;" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_000913.3?report=fasta&to=4641652">https://www.ncbi.nlm.nih.gov/nuccore/NC_000913.3?report=fasta&to=4641652</a>.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="image2">
          <b>[Image 6]</b> 
          <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_000913.3?report=fasta&to=4641652" style="font-weight:450;">https://www.ncbi.nlm.nih.gov/nuccore/NC_000913.3?report=fasta&to=4641652
        </a>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="cite2">
          <b>[2]</b> Ursell, L. K., Metcalf, J. L., Parfrey, L. W., & Knight, R. (2012). Defining the human microbiome. <i>Nutrition reviews</i>, 70 Suppl 1(Suppl 1), S38–S44. <a target="_blank" style="font-weight:450;" href="https://doi.org/10.1111/j.1753-4887.2012.00493.x">https://doi.org/10.1111/j.1753-4887.2012.00493.x</a>.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="cite2">
          <b>[Image 7]</b>
          <a target="_blank" style="font-weight:450;" href="http://qiime.org/">http://qiime.org/</a>.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="cite2">
          <b>[Image 8]</b>
          <a target="_blank" style="font-weight:450;" href="https://mothur.org/">https://mothur.org/</a>.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12" id="cite3">
          <b>[3]</b> Goodrich K., J et al. (2014). Leading Edge Primer. <i>Conducting a Microbiome Study</i>. <a target="_blank" style="font-weight:450;" href="https://dx.doi.org/10.1016%2Fj.cell.2014.06.037">https://dx.doi.org/10.1016%2Fj.cell.2014.06.037</a>.
      </div>
      
  </div>
</div>
</div>
  
<!-- end abouts -->
<footer>
  <br>
  <div class="footer">
    <div class="container">
      <div class="row">
        <div class="col-xl-4 col-lg-4 col-md-4 col-sm-12">
          <div class="Contact">
            <h3>
              <a href="sources.html" style="color:white;">
                Resources
              </a>
            </h3>
            <!--
            <ul class="contant_icon">
              <li> <a href="mailto:fascettij21@bcdschool.org"><img src="icon/email.png"></a></li>
            </ul>
            -->
          </div>
        </div>
        <div class="col-xl-4 col-lg-4 col-md-4 col-sm-12">
          <div class="Contact">
            <h3>
              <a href="glossary.html" style="color:white;">
                Glossary
              </a>
            </h3>
            <!--
            <ul class="contant_icon">
              <li> <a href="mailto:fascettij21@bcdschool.org"><img src="icon/email.png"></a></li>
            </ul>
            -->
          </div>
        <!--
        <div class="col-xl-3 col-lg-3 col-md-3 col-sm-12">
          <div class="Social">
            <h3>Social links</h3>
            <ul class="socil_link">
              <li><a href="#"><img src="icon/fb.png"></a></li>
              <li><a href="#"><img src="icon/Tw.png"></a></li>
              <li> <a href="#"><img src="icon/lin.png"></a></li>
              <li> <a href="#"><img src="icon/insta.png"></a></li>
            </ul>
          </div>
        </div>
        <div class="col-xl-5 col-lg-5 col-md-5 col-sm-12">
          <div class="newsletter">
            <h3>newsletter</h3>
            <input class="new" placeholder="Enter your email" type="Enter your email" >
            <button class="subscribe">subscribe</button>
          </div>
        </div>
        -->
      </div>
    </div>
    Jonathan Fascetti, 2020.
  </div>
  <!--
  <div class="copyright">
    <div class="container">
      <p>Copyright 2019 All Right Reserved By <a href="http://html.design">Free html Templates</a></p>
    </div>
  </div>
  -->
</footer>
<!-- end footer --> 

<!-- Javascript files--> 
<script src="js/jquery.min.js"></script> 
<script src="js/popper.min.js"></script> 
<script src="js/bootstrap.bundle.min.js"></script> 
<script src="js/jquery-3.0.0.min.js"></script> 
<script src="js/plugin.js"></script> 

<!-- sidebar --> 
<script src="js/jquery.mCustomScrollbar.concat.min.js"></script> 
<script src="js/custom.js"></script>
</body>
</html>